feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radio_nl80211.c

```diff
@@ -479,11 +505,12 @@ static void vif_poll_stations(void *arg)
 	avl_for_each_element(&wif_tree, wif, avl) {
 		struct nl_msg *msg;
 
+		remove_stale_station(wif->name);
```

Comment:
The call to remove_stale_station() could be done after the unl_genl_request().  This would avoid "deletion and then immediate reinsertion" corner case.

Reply:
Done.